config/routes.rb

```diff
@@ -38,6 +38,7 @@
             end
           end
         end
+        resources :docker_registries, :except => [:new, :edit]
```

Comment:
Since the API is namespaced, (`docker/api/v2/`) does saying again `*docker*_registries` serves any purpose vs just `registries`?